lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -86,11 +86,11 @@ def self.init_vmpooler()
         execute_rake_task("beaker_quickstart:gen_hosts[vmpooler]")
       end
 
-      # Print a message to the console and exit with 0
+      # Print a message to the console and exit with 1
       # @param [String] msg the message to print
       def self.exit_with(msg)
```

Comment:
Should we make the exit code a parameter of this method?  And have it default to `1`?

I haven't looked at the usages of this method, or the JIRA ticket, but just looking at this method it sounds like it wants to be a general-purpose method for printing and exiting, and we've already got the message parameterized.